contracts/MultiVesting.sol

```diff
@@ -10,12 +10,13 @@ contract MultiVesting is IVesting, Ownable {
     using SafeERC20 for IERC20;
 
     event SetSeller(address newSeller);
-    event Vested(address beneficiary, uint256 amount);
+    event Vested(address indexed beneficiary, uint256 amount);
     event EmergencyVest(uint256 amount);
-    event UpdateBeneficiary(address oldBeneficiary, address newBeneficiary);
+    event UpdateBeneficiary(address indexed oldBeneficiary, address indexed newBeneficiary);
     event DisableEarlyWithdraw(address owner);
 
     IERC20 public immutable token;
+    uint256 public sumVesting;
     address public seller;
     address public constant GNOSIS = 0x42DA5e446453319d4076c91d745E288BFef264D0;
```

Comment:
"L03. Code quality

In the project, there are hardcoded variables GNOSIS, that are marked as constant. We recommend avoiding using hardcoded addresses and marking them as immutable in order to make the testing process easier."